nau_openedx_extensions/filters/models.py

```diff
@@ -0,0 +1,23 @@
+"""Models used in openedx filters."""
+
+from django.db import models
+from opaque_keys.edx.django.models import CourseKeyField
+
+
+class DomainsAllowedPerCourse(models.Model):
```

Comment:
I am more in favour of putting this into the [advanced course settings](https://github.com/openedx/edx-platform/blob/d9f54b9f460bed0001e999dac8884d3b8e8a949f/xmodule/course_module.py#L1021) in studio. It prevents the need for a custom model in mysql. You can access this via studio and therefore you can make the course authors use it.

@igobranco @sandroscosta  what would be your preference here? I think we have used "other_course_settings" in the past with Nau

Reply:
@felipemontoya  and @MaferMazu  I didn't know the existence of the "other_course_settings" field.
We don't have it activated on NAU.
I have the same opinion @felipemontoya I think it is better to use "other_course_settings" on the Course Studio advanced settings.